package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "lint": "eslint bin lib test examples client-src",
     "test": "nyc --reporter lcovonly mocha --full-trace --check-leaks --exit",
-    "prepare": "(rm ssl/*.pem || true) && npm run -s transpile:index && npm run -s build:live && npm run -s build:index && npm run -s build:sockjs",
+    "prepare": "./node_modules/.bin/rimraf ./ssl/*.pem && npm run -s transpile:index && npm run -s build:live && npm run -s build:index && npm run -s build:sockjs",
```

Comment:
```
"prepare": "rimraf ./ssl/*.pem && ..."
```

`npm` should look up `node_modules/.bin` automatically or didn't this work for you (on windows)?

Reply:
On Windows it doesn't automatically look into that folder

```console
c:\path\webpack-dev-server>.\node_modules\.bin\rimraf ssl/*.pem

c:\path\webpack-dev-server>rimraf ssl/*.pem
'rimraf' is not recognized as an internal or external command,
operable program or batch file.
```